source/common/mongo/proxy.h

```diff
@@ -73,6 +77,31 @@ class AccessLog {
 typedef std::shared_ptr<AccessLog> AccessLogSharedPtr;
 
 /**
+ * Mongo fault configuration.
+ */
+class FaultConfig {
```

Comment:
That's correct, I did not say there is no way to test it.

This is to avoid relying on json for testing filter functionality, which inherently introduces additional code complexity.
Going to stick to less code == fewer bugs for now.